internal/pkg/service/stream/storage/level/local/encoding/encoder/csv/fastcsv/writer.go

```diff
@@ -87,43 +85,3 @@ func (w *writer) WriteRow(cols *[]any) error {
 
 	return nil
 }
-
-func (w *writer) toCSVValue(v any) (string, error) {
-	// Inspired by cast.ToStringE(), but without slow reflection
-	switch s := v.(type) {
-	case string:
```

Comment:
We need it now also in the API mapper, so I moved it as public to the `csvfmt` pkg.